values.yaml

```diff
@@ -41,9 +41,9 @@ ingress:
     # apiPath: /api
     # uiPath: /ui
     # storePath: /store
-    apiPath: ""
-    uiPath: ""
-    storePath: ""
+    apiPath: ":9001"
+    uiPath: ":9000"
+    storePath: ":9002"
```

Comment:
This is little tricky, but you can set port. Please check [here](https://github.com/yoshwata/screwdriver-chart/blob/9c3b22f09955da18263d72ec05a622925fac09b4/templates/_helpers.tpl#L41).